.github/workflows/ci.yml

```diff
@@ -82,7 +82,7 @@ jobs:
           function _make() { make -j$(nproc) -s O=out $*; }
           function download_plug_and_trust() { mkdir -p $HOME/se050 && git clone --single-branch -b v0.4.2 https://github.com/foundriesio/plug-and-trust $HOME/se050/plug-and-trust || (rm -rf $HOME/se050 ; echo Nervermind); }
 
-          function download_scp_firmware() { git clone --single-branch https://github.com/ARM-software/SCP-firmware.git $HOME/scp-firmware || echo Nervermind; }
+          function download_scp_firmware() { git clone -n --single-branch https://github.com/ARM-software/SCP-firmware.git $HOME/scp-firmware && git -q -C $HOME/scp-firmware checkout v2.13.0 || echo Nervermind; }
```

Comment:
In fact I believe the simplest/fastest/most efficient command would be `git clone --depth 1 https://github.com/ARM-software/SCP-firmware.git -b v2.13.0`

Reply:
`-b` works on a tag revision? If so, yes, it's preferable.

Reply:
I'll update. The cmdline I wrote is bugged, fails on git checkout argument `-q` (not available on `git checkout`).